src/searchPage/searchPageClient.spec.ts

```diff
@@ -1601,6 +1602,23 @@ describe('SearchPageClient', () => {
         expectMatchDescription(built.description, SearchPageEvents.generatedAnswerCollapse, meta);
     });
 
+    it('should send proper payload for #logGeneratedAnswerFeedbackSubmiV2', async () => {
```

Comment:
```suggestion
    it('should send proper payload for #logGeneratedAnswerFeedbackSubmitV2', async () => {
```
i think you're missing a "t" here